app/templates/components/job-authorize.hbs

```diff
@@ -1,10 +1,9 @@
-<div class="col-md-12">
-  {{#bs-form as |form|}}
-    {{#form.group class='job-authorize-group form-inline pull-right' }}
-        <label for="authorize-job">Enter job token:</label>
-        {{input class='form-control job-name-input' id='authorize-job' value=token}}
-        {{#bs-button type='primary' onClick=(action "authorize")}}Authorize{{/bs-button}}
-    {{/form.group}}
-  {{/bs-form}}
-</div>
-{{yield}}
+{{#bs-form as |form|}}
+  {{#form.group class='job-authorize-group form-inline' }}
+      <label for="authorize-job">Authorization Code:</label>
+      {{input class='form-control job-name-input' id='authorize-job'
+              value=token
+              disabled=job.hasAuthorization}}
+      {{#bs-button type='primary' onClick=(action "authorize") disabled=job.hasAuthorization}}Authorize{{/bs-button}}
```

Comment:
If a user enters a bogus authorization code the reason isn't displayed.
The response back from bespin-api looks like this: 
```
{"errors":[{"status":400,"detail":"This is not a valid token."}]}
```
Fixing this may require bespin-api changes to fit with the ember data model.
This may have already been broken and is outside of the scope of this PR.

Reply:
> duplication for authorization code when state is Authorized

This was intentional. I felt that it made sense to keep the input field around when the job goes from new->authorized to ease the transition, but also to put the authorization code on the right side whenever we have it. I'd like to go ahead with this change and see what feedback we get (if any). I wouldn't expect jobs to stay in `Authorized` for very long.

> Also the Finished step says "Your job has completed. Results are available at the location below:
/"
> I thought it should say something about the results being reviewed by the share group.

Yeah, there are two issues here. One is that the job-output-dir model changed (#70) so the text of the link here is useless. Also, since we're no longer giving access directly to the user immediately, they won't have access to the uploaded project, and may only have access to a copy. Yes it needs to change, but that warrants more discussion so I'll create an issue

> If a user enters a bogus authorization code the reason isn't displayed.

I'll add an issue for this. Should be workable without changes to bespin-api